handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslServerContext.java

```diff
@@ -67,13 +67,35 @@ private ReferenceCountedOpenSslServerContext(
             Iterable<String> ciphers, CipherSuiteFilter cipherFilter, OpenSslApplicationProtocolNegotiator apn,
             long sessionCacheSize, long sessionTimeout, ClientAuth clientAuth, String[] protocols, boolean startTls,
             boolean enableOcsp) throws SSLException {
+        this(trustCertCollection, trustManagerFactory, keyCertChain, key, keyPassword,
```

Comment:
This is within a private constructor. Just modify the callers of the private constructor to pass the new parameter. Ditto for package-private constructor.